### PR TITLE
redesign of teacher homepage

### PR DESCRIPTION
**Pull request by tanyaparker**
**DONE:**
- move miniblock css from pegasus/dashboard -> shared
- add sections of additional courses: hoc, k5, middle, high school
- extend k5 course blocks to long description
- middle and high school courses lead to curriculum pages
- add a footer for more tutorials -> /learn/beyond
- update 20-hour course messaging to "Accelerated Intro to CS Course"
- fix beta yellow

**BEFORE:**
image

**AFTER:**
image

### CODE REVIEW

File: dashboard/app/views/api/courses.html.haml [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+
+%br/
+
+.row
+  .span12.twolineheader!= t('home.heading_legacy')
+.row
+  = render partial: 'shared/course_wide_block', locals: { id: '20hour' }
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (breville) [inline dashboard/app/views/api/courses.html.haml]: is this common with _all_courses.html.haml?

File: pegasus/sites.v3/code.org/public/css/page.css [hunk 2]
```diff
@@ -42,107 +42,3 @@
...
-.miniblock a:hover,
-.miniblock a:active {
-  color: #fff;
-}
-
-/* End of duplicated from dashboard for included mini blocks. */
-
-/* Additional styling to make mini blocks work. */
-
-.span1 {  width: 60px;  }
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/page.css]: trying to think whether there's anywhere else in pegasus that might rely on this.  probably none, but worth pondering.

**Comment by breville** [comment]
great refactoring!

**Comment by tanyaparker** [comment]
@breville Ok now ready for first look.

File: shared/css/course-blocks.scss [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+.betatext {
+  color: $realyellow;
+  background-color: $charcoal;
+  padding: 0 5px;
+}
+
+/* MINIBLOCKS */
+
+.span1 {  width: 60px;  }
```
Comment (breville) [inline shared/css/course-blocks.scss]: oh, doesn't this possibly compete with bootstrap's own definitions of these spans?  can we namespace them so they don't compete?

**Pull request merged**